src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -17,11 +17,13 @@
 
 import grpc
 
-from grpc_health.v1 import health_pb2
-from grpc_health.v1 import health_pb2_grpc
+from grpc_health.v1 import health_pb2 as _health_pb2
+from grpc_health.v1 import health_pb2_grpc as _health_pb2_grpc
```

Comment:
Why are we making this change?

Reply:
When you import a symbol without underscore, it'll be available as a "public" (non-underscored) property of the module and therefore looks like it's part of your public API, which is not the intnet. The symbols imported here should not be publicly available.